packages/tc-ui/src/pages/edit/__tests__/e2e-edit-MVR-record.cyp.js

```diff
@@ -0,0 +1,77 @@
+const { code, someString } = require('../../../test-helpers/mainTypeData.json');
```

Comment:
I don't think we need this as well as the edit record tests for MainType. Are you testing anything different here compared to the MainType one?

Reply:
it was only `it('can not edit code of record', () => { ...}` will move that test to MainType